### DISCUSSION TITLE
Miyoo Mini

### DISCUSSION THREAD

**Discussion by jtothebell** [jtothebell/fake-08#100] [General]
Curious if there is interest in a Miyoo Mini build. I don't have a device, but I was able to make a build for it using https://github.com/MiyooMini/union-toolchain.  Based on things I read, I have what I think is the correct surface types (32 bpp, hardware accelerated) and rom path, and keys, but it is untested. The changes are published in this branch: https://github.com/jtothebell/fake-08/tree/MiyooMini
[FAKE08-MiyooMini.zip](https://github.com/jtothebell/fake-08/files/7964757/FAKE08-MiyooMini.zip)

**Comment by jtothebell** [comment]
There were problems with that initial build. It might be fixed with this: 
[FAKE08-MiyooMini-01.zip](https://github.com/jtothebell/fake-08/files/7964828/FAKE08-MiyooMini-01.zip)

**Comment by jtothebell** [comment]
This one should really be playable
[FAKE08-MiyooMini-02.zip](https://github.com/jtothebell/fake-08/files/7964940/FAKE08-MiyooMini-02.zip)

**Comment by jtothebell** [comment]
Swap function and select buttons, and tweak audio settings to try help with the delay
[FAKE08-MiyooMini-03.zip](https://github.com/jtothebell/fake-08/files/7965066/FAKE08-MiyooMini-03.zip)

**Comment by jtothebell** [comment]
This build should allow loading png carts with a .p8 extension (and vice versa)
[FAKE08-MiyooMini-04.zip](https://github.com/jtothebell/fake-08/files/7965101/FAKE08-MiyooMini-04.zip)

**Comment by jtothebell** [comment]
trying miyoo ao audio implementation
[FAKE08-MiyooMini-05.zip](https://github.com/jtothebell/fake-08/files/7967216/FAKE08-MiyooMini-05.zip)

**Comment by jtothebell** [comment]
Here are some tweaks mostly from @ eggs on the Retro Game Handhelds discord, who generously helped figure out a lot of the sound issues, along with the clipping audio on all platforms. Still untested by me, but sound latency should be better. Also a lot smaller because it doesn't have debug symbols.
[FAKE08-MiyooMini-06.zip](https://github.com/jtothebell/fake-08/files/7968153/FAKE08-MiyooMini-06.zip)

**Comment by jtothebell** [comment]
This only has a couple tweaks, and should be included in the next release for all platforms shortly
[FAKE08-MiyooMini.zip](https://github.com/jtothebell/fake-08/files/8100926/FAKE08-MiyooMini.zip)

**Comment by bigmonkeynads** [comment]
Keen for more updates/builds if there is any? Getting a lot of “no lua” errors when trying to play Pico8 on Miyoo Mini. It’s more the newer more current games

**Reply by jtothebell** [reply]
The latest Miyoo mini builds are in the releases section now with all the other platforms. If you are running the latest release and still getting errors, please create and issue with the specific cart linked.

**Reply by bigmonkeynads** [reply]
Ah ok that’s great thank you, I’ll check it out soon really appreciate all that you do for the Miyoo Mini for Pico8

**Comment by tarikalmashni** [comment]
Thank you sooooo much for this, really. It lifts the Miyoo Mini even beyond where it was without PICO-8.

**Comment by edgarlim** [comment]
Thank you @jtothebell for creating the emulator. It has been a blast playing it on my Miyoo Mini! Really hope you are able to somehow add a "save state" feature to the emulator, if it is possible that will be awesome!

**Comment by thediceabide** [comment]
Does anyone have issue with screen flicker on some games? Specifically the black bars around the game. From what I've tested so far: Porklike, Orul, Tetris don't flicker, Celeste, Matchy Matchy, Just One Boss and Swordfish all flicker. Furthermore I do not get any flicker in the menu screen regardless of which game.
